include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
There are ways to use pthread on windows and that's probably the easiest way to get current code working on windows. However, if Window's native threading API is to be used (and wrapped in pthread API),  it would be less confusing with defining these types directly(instead of "struct") in the os dependent headers.